src/Containers/ImageCarousel.tsx

```diff
@@ -8,6 +8,8 @@ import { ImplicitPropsInterface } from '@Utils/Hooks';
 export interface ImageCarouselProps
     extends MainInterface,
         ResponsiveInterface,
+        // doesn't work for some reason - havent been able to find a solution
```

Comment:
Might be the onClick prop conflicting with it, remove it and see what happens?

Reply:
@fpunny But dont you need the custom onClick prop because it returns the image? If I remove the onClick you won't be able to click on the image to delete it

Reply:
https://stackoverflow.com/questions/41285211/overriding-interface-property-type-defined-in-typescript-d-ts-file

KK then try one of these solutions which omit's the property so that it can be made custom

Reply:
Omit works! @fpunny